declarations/momentum/PSAR.d.ts

```diff
@@ -0,0 +1,15 @@
+import { IndicatorInput, Indicator } from '../indicator/indicator';
+export declare class PSARInput extends IndicatorInput {
+    step: number;
```

Comment:
Maybe could have a `start` like trading view? the start here is obviously the step increment only